packages/graphile-build-pg/src/plugins/PgMutationCreatePlugin.js

```diff
@@ -44,6 +44,11 @@ export default (function PgMutationCreatePlugin(
           .filter(table => !!table.namespace)
           .filter(table => table.isSelectable)
           .filter(table => table.isInsertable)
+          .filter(
+            table =>
+              (pgEnableDefaultMutationTables.length == 0) ||
```

Comment:
Since the option above has no default, this line will throw an error if the caller doesn't specify the option.

Reply:
Oh, I misunderstood the option was set default value here.
https://github.com/postgraphql/postgraphql/pull/595/commits/0cefb2bf6809c6adcb345b6e69368c5d442b7444#diff-6c38b9313e10fd19f30ddbdd09c18c73R87

Where should I set the default value? Around here?
https://github.com/graphile/graphile-build/pull/86/commits/b4fbf693c1639003ef8110e029aeaae0d9085c0d#diff-886ba16c8a211054c79f05dbd24bac7fR14